src/searchForm/SearchForm.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useContext } from 'react'
+import cc from 'classcat'
+
+import { color } from '_utils/branding'
+import SearchIcon from 'icon/searchIcon'
+import { CalendarIcon } from 'icon/calendarIcon'
+import { Bullet } from 'index'
+import { BulletTypes } from 'bullet'
+import { StandardSeat } from 'icon/standardSeat'
+import { DoubleArrowIcon } from 'icon/doubleArrowIcon'
+import Divider from 'divider'
+import { MediaSize, MediaSizeContext } from '_utils/mediaSizeProvider/MediaSizeProvider'
+
+export interface SearchFormProps {
```

Comment:
Only `className` is used for now.
I think you should remove other vars since you're not usign them